packages/vite-app-ts/.eslintrc

```diff
@@ -59,12 +62,7 @@
         "additionalHooks": "(useRecoilCallback)"
       }
     ],
-    "prettier/prettier": [
-      "warn",
-      {
-        "endOfLine": "auto"
-      }
-    ],
+    "prettier/prettier": "error",
```

Comment:
rever these changes, prettier will auto fix in ide and lining, there's no reason for this to be errors